docs/approvals/service-now/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Service Now Integration
+description: Approvals, approvals, approvals
+position: 10
+---
+:::warning
+The ServiceNow Integration feature is only available as an EAP for version 2022.2.5556 or later and it requires a feature license to be able to use it.
+:::
+
+## Overview
+
+The Octopus Deploy/ServiceNow integration allows users to block the execution of specifically configured deployments unless they have a corresponding approved ServiceNow **Change Request** (CR).
+
+For configured project and environment combinations, Octopus Deploy will 
+
+To enable this behavior, both systems must be configured before deployments can be managed.
+
+| Project | Environment Change Controlled | Environment **Not** Change Controlled| 
```

Comment:
Instead of a matrix, what if the table show the 3 possible combinations and the expected outcome.

| Project | Environment | Outcome |
| -- | -- |  -- |
| Change controlled | Change controlled | **Approval required for deployment** |
| _Not_ change controlled | Change controlled | No approval required |
| Change controlled | _Not_ change controlled | No approval required |

Reply:
I have used this matrix as it readable without having to jump through hoops